adding-module.rst

```diff
@@ -10,6 +10,7 @@ will need to extend the existing firmware with your own C code, and this
 article is going to show you how to do it, using the ESP8266 port as the
 example.
 
+Note: you can use any naming-scheme you like and do not have to name your variables or data-type in any specific way. 
```

Comment:
The fact that you can doesn't mean you should. Consistency with the existing code is important.